clang/lib/Driver/Driver.cpp

```diff
@@ -87,6 +87,7 @@
 #include "llvm/Support/Process.h"
 #include "llvm/Support/Program.h"
 #include "llvm/Support/RISCVISAInfo.h"
+#include "llvm/Support/Regex.h"
```

Comment:
remove #include "llvm/Support/Regex.h" for clang/tools/driver/driver.cpp

Reply:
thanks for catching that